website/docs/api/regions.mdx

```diff
@@ -0,0 +1,239 @@
+---
+id: regions
+title: Region functions
+sidebar_label: Regions
+slug: /api/regions
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+These functions convert H3 indexes to and from polygonal areas.
+
+## polyfill
+
+<Tabs
+  groupId="language"
+  defaultValue="c"
+  values={[
+    {label: 'C', value: 'c'},
+    {label: 'Python', value: 'python'},
+    {label: 'Java', value: 'java'},
+    {label: 'JavaScript (Live)', value: 'javascript'},
+  ]
+}>
+<TabItem value="c">
+
+```c
+void polyfill(const GeoPolygon* geoPolygon, int res, H3Index* out);
+```
+
+</TabItem>
+<TabItem value="python">
+
+```py
+h3.polyfill(polygons, res)
+```
+
+</TabItem>
+<TabItem value="java">
+
+```java
+List<Long> polyfill(List<GeoCoord> points, List<List<GeoCoord>> holes, int res);
+List<String> polyfillAddress(List<GeoCoord> points, List<List<GeoCoord>> holes, int res);
+```
+
+</TabItem>
+<TabItem value="javascript">
+
+```js
+h3.polyfill(polygon, res)
+```
+
+```js live
+function example() {
+    const polygon = [
+        [37.813318999983238, -122.4089866999972145],
+        [37.7198061999978478, -122.3544736999993603],
+        [37.8151571999998453, -122.4798767000009008]
+    ];
+    const res = 7;
+    return h3.polyfill(polygon, res);
+}
+```
+
+</TabItem>
+</Tabs>
+
+polyfill takes a given GeoJSON-like data structure and preallocated,
+zeroed memory, and fills it with the hexagons that are contained by
+the GeoJSON-like data structure.
+
+The current implementation is very primitive and slow, but correct,
+performing a point-in-poly operation on every hexagon in a k-ring defined
+around the given geofence.
```

Comment:
Is this still true? I thought @nrabinowitz replacement algorithm was merged and not reverted?

Reply:
The text (not the API definition) wasn't updated from the previous yet.